code/modules/client/preference_setup/loadout/lists/headwear.dm

```diff
@@ -21,9 +21,9 @@
 	path = /obj/item/clothing/head/beret/purple
 
 /datum/gear/head/beret/bsec
-	display_name = "beret, navy (officer)"
+	display_name = "beret, navy"
 	path = /obj/item/clothing/head/beret/sec/navy/officer
-	allowed_roles = list("Ironhammer Operative","Ironhammer Commander","Gunnery Sergeant")
+	allowed_roles = list("Ironhammer Operative","Ironhammer Commander","Gunnery Sergeant",JOBS_MEDICAL)
```

Comment:
![изображение](https://user-images.githubusercontent.com/57810301/118983872-91a17a80-b985-11eb-8750-39e677e73f8c.png)
This one is directly security one as it follows IH palette. If you want a moebius beret so badly make a different one rather than making them share with IH.
```suggestion
	allowed_roles = list("Ironhammer Operative","Ironhammer Commander","Gunnery Sergeant")
```

Reply:
I had a miscommunication with CeUvi, am fixing